zebedee-reader/src/main/java/com/github/onsdigital/zebedee/reader/api/endpoint/PublishedIndex.java

```diff
@@ -0,0 +1,44 @@
+package com.github.onsdigital.zebedee.reader.api.endpoint;
+
+import com.github.davidcarboni.restolino.framework.Api;
+import com.github.onsdigital.zebedee.reader.api.bean.PublishedIndexResponse;
+import com.github.onsdigital.zebedee.search.indexing.Document;
+import com.github.onsdigital.zebedee.search.indexing.FileScanner;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.GET;
+import java.io.IOException;
+import java.util.List;
+
+@Api
+public class PublishedIndex {
```

Comment:
There is no unit test to cover the functionality added by this class. Ideally We should not be putting new functionality live that has not been tested.

Reply:
basic unit test included in existing test scenario.